### ISSUE TITLE
activity ?

### ISSUE THREAD

**Issue by jqueguiner** [criteo/lobster#6]
is it still active ?

**Comment by brugidou** [comment]
I don't think that there are any planned features but if you have an issue with the tool I can have a look.
